src/sensor.rs

```diff
@@ -447,4 +449,32 @@ where
         let gyro = Gyro::from_bytes([data[8], data[9], data[10], data[11], data[12], data[13]]);
         Ok((accel, gyro))
     }
+
+    /// Read the current temperature from the sensor's internal temperature sensor.
+    ///
+    /// Note: This measures the internal temperature of the MPU-6050 chip itself,
```

Comment:
slightly duplicated docs here. I believe the docs for the Temperature struct below are sufficient and a good place for the note about ambient vs. device temp.